### PR TITLE
 [stable/moodle] Fix some typos

### PR DESCRIPTION
**Pull request by carrodher**

### CODE REVIEW

**Comment by carrodher** [comment]
Re-reviewed

File: stable/moodle/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Moodle
 
-[Moodle](https://www.moodle.org) is a learning platform designed to provide educators, administrators and learners with a single robust, secure and integrated system to create personalised learning environments
+[Moodle](https://www.moodle.org) is a learning platform designed to provide educators, administrators and learners with a single robust, secure and integrated system to create personalized learning environments
```
Comment (sameersbn) [inline stable/moodle/README.md:3]: :slightly_smiling_face:

**Pull request merged**